### DISCUSSION TITLE
Initialization options are not preserved with setContent method

### DISCUSSION THREAD

**Discussion by vlad-kuz** [ueberdosis/tiptap#5933] [General Questions]
We using angular and creating editor with `parseOptions: {preserveWhitespace: 'full'}`:
```typescript
  editorEl = viewChild.required<ElementRef<HTMLDivElement>>('editorEl');
  tiptapEditor = new Editor({
    content: '<p>a    b</p>',
    parseOptions: {
      preserveWhitespace: 'full',
    },
    extensions: [StarterKit.configure()],
  });

  ngAfterViewInit(): void {
    this.editorEl().nativeElement.append(...Array.from(this.tiptapEditor.options.element.childNodes));
    this.tiptapEditor.setOptions({element: this.editorEl().nativeElement});
  }
```
after that content is displayed correctly:
```html
<div contenteditable="true" role="textbox" translate="no" class="tiptap ProseMirror" tabindex="0">
  <p>a    b</p>
</div>
```
but when we using **setContent** to change this conent (without passing third parameter ***parseOptions***):
```typescript
  this.tiptapEditor.chain().setContent('<p>a    b</p>', false).run();
```
editor remove spaces:
```html
<div contenteditable="true" role="textbox" translate="no" class="tiptap ProseMirror" tabindex="0">
  <p>a b</p>
</div>
```
if we call **setConent** with ***parseOptions***
```typescript
this.tiptapEditor.chain().setContent('<p>a    b</p>', false, {preserveWhitespace: 'full'}).run();
```
spaces are preserved, as it was after initialization:
```html
<div contenteditable="true" role="textbox" translate="no" class="tiptap ProseMirror" tabindex="0">
  <p>a    b</p>
</div>
```

**Question**. This is expected behavior? We should every time pass parseOptions with method setContent and options passed during initialization is only used to initial content?
In documentation (https://tiptap.dev/docs/editor/api/commands/content/set-content#parameters):

> parseOptions?: Record<string, any> - Options to configure the parsing can be passed during initialization and/or with setContent.

**Comment by liferays** [comment]
any updates ?
